diff --git a/node_modules/frappe-ui/frappe/Help/HelpModal.vue b/node_modules/frappe-ui/frappe/Help/HelpModal.vue
index 3e17c76..062fe98 100644
--- a/node_modules/frappe-ui/frappe/Help/HelpModal.vue
+++ b/node_modules/frappe-ui/frappe/Help/HelpModal.vue
@@ -36,7 +36,7 @@
         :appName="appName"
       />
       <HelpCenter
-        v-else-if="showHelpCenter"
+        v-else-if="false"
         v-model="articles"
         :docsLink="docsLink"
       />
@@ -139,7 +139,7 @@ const footerItems = computed(() => {
         syncStatus()
         showHelpCenter.value = true
       },
-      condition: !isOnboardingStepsCompleted.value && !showHelpCenter.value,
+      condition: false,
     },
     {
       icon: StepsIcon,
